plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -23,6 +23,7 @@ const config = {
 	retries: CI ? 2 : 0,
 	repeatEach: REPEAT_EACH ? Number( REPEAT_EACH ) : 1,
 	workers: 1,
+	reportSlowTests: { max: 5, threshold: 30 * 1000 }, // 30 seconds threshold
```

Comment:
I picked some somewhat arbitrary numbers here, we can adjust.  What this says is that a test will be reported as slow if it takes longer than 30 seconds.  The max number is that it's reporting the 5 slowest tests per shard... having a list of more than 25 tests probably isn't super productive.